modules/ROOT/pages/_partials/sync/creating-a-voyager-server-dockerfile-for-use-with-openshift.adoc

```diff
@@ -0,0 +1,61 @@
+[id="creating-a-voyager-server-dockerfile-for-use-with-openshift-{context}"]
```

Comment:
this (and the file name) are now inaccurate I think: it's not about creating a Dockerfile, but rather a container image.